setup.py

```diff
@@ -16,7 +16,7 @@ def package_data(pkg, root):
 
 setup(
     name='recommender-xblock',
-    version='0.1',
+    version='0.2',
```

Comment:
I believe the format for the version needs to match the format used in the actual release tag: https://github.com/pmitros/RecommenderXBlock/releases/tag/v1.0